### DISCUSSION TITLE
What param should I pass to get products of US

### DISCUSSION THREAD

**Discussion by Vishnu-Jmatics** [openfoodfacts/openfoodfacts-dart#454] [Q&A] [answered]
In 'OpenFoodAPIClient.searchProducts()' by keyword what param should I pass to get products of US only?

**Comment by monsieurtanuki** [comment, marked as answer]
```dart
  ProductSearchQueryConfiguration({
    final OpenFoodFactsLanguage? language,
    final List<OpenFoodFactsLanguage> languages = const [],
    final OpenFoodFactsCountry? country,
    final List<ProductField>? fields,
    required List<Parameter> parametersList,
  })
```

`final OpenFoodFactsCountry? country` seems appropriate.
Another solution would be to set `OpenFoodAPIConfiguration.globalCountry`.

**Comment by Vishnu-Jmatics** [comment]
@monsieurtanuki Thanks. It works fine.

**Reply by monsieurtanuki** [reply]
@Vishnu-Jmatics Cool! Please mark it as a correct answer.
